nslsii/tests/temperature_controllers_test.py

```diff
@@ -26,14 +27,22 @@ def test_Eurotherm(RE):
     stdin = None
 
     # Start up an IOC based on the thermo_sim device in caproto.ioc_examples
+    '''
```

Comment:
please do not comment things out by putting strings around them, it makes the diff very hard to read.  Either remove the old code (it is under version control so we can get it back!) or put `#` at the beginning of each line (most editors have helpers to do this in bulk).